references/video_classification/train.py

```diff
@@ -164,7 +164,7 @@ def main(args):
 
     if args.cache_dataset and os.path.exists(cache_path):
         print(f"Loading dataset_train from {cache_path}")
-        dataset, _ = torch.load(cache_path, weights_only=True)
+        dataset, _ = torch.load(cache_path)
```

Comment:
Thanks for the PR @Met4physics ,  I think the default of weights_only recently changed to `None`, so we might need to keep it hard-coded as `weights_only=False`

similar to what we do in the classification reference: https://github.com/pytorch/vision/blob/acbfd8d94d10f989f4540252e92e8855c19f7ff7/references/classification/train.py#L131


Do you mind updating this PR?

Reply:
Thank you for the feedback! I have already updated this PR.